src/config.js

```diff
@@ -360,6 +360,7 @@ const ScopeDescriptor = require('./scope-descriptor')
 // * Don't depend on (or write to) configuration keys outside of your keypath.
 //
 const schemaEnforcers = {}
+const PROJECT = '__atomproject__'
```

Comment:
Could we just use the actual project file path as the `source`?

Reply:
That was my original implementation, but what if in the future configs sourced to individual files becomes common? Could that lead to collisions? 

Maybe I could do something like `project__${source}`?